recipes-core/curl/curl_%.bbappend

```diff
@@ -0,0 +1,3 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
```

Comment:
I think FILESEXTRAPATHS_prepend is not necessary, you don't add any file

Reply:
Indeed, fixed